README.md

```diff
@@ -13,8 +13,9 @@ This document outlines guidelines that we believe are necessary to encourage con
 * Standards discussions should adhere to standards organizations’ **Codes of Conduct**
 * Participants should follow some simple **golden rules for standards discourse**
     * Be respectful, and recognize the validity of others’ points of view
-    * Don’t assume, ask for confirmation instead
+    * Don’t assume, ask for confirmation instead.  Look for (and communicate) motivations, rather than surface details.
```

Comment:
This seems like the opposite of what we want, actually. I'd like to discuss proposals, not debate people's motivations.

Reply:
Hmm.  I agree "bring specific proposals" might belong in there; but many times, it seems that we end up bikeshedding because we are not clear on WHY the bikeshed needs to be a light color (because it reflects heat better). I think when participants are more clear about what their needs are at their roots, it is easier to solve a solution collaboratively.

Does that seem appropriate, and I just chose poor words, or do you disagree with that expanded sentiment?

Reply:
I think my problem here is that term "motivations" rather than requirements.

What I want to avoid is "you really want feature X because you want <objectionable thing Y>"

Reply:
"Look for (and communicate) basic requirements."  ?

I think I see what you mean, and I agree I'd like to avoid "you really want feature X because" - but that's because I want it to be more common to state it as "I think I need feature X because I need to be able to do Y."

Reply:
Yah, that text looks better.

Reply:
"motivations" (or "incentives") seems too abstract/vague to be actionably useful here. "need to be able to do Y" captures the sentiment better. I'd prefer something like "use-case" or "problem statement"

Reply:
updated below, PTAL.